api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/manager/EptsDataExportManager.java

```diff
@@ -76,8 +74,7 @@ public ReportDesign createExcelDesignWithProperties(String reportDesignUuid, Rep
 	 * @return
 	 * @throws IOException
 	 */
-	public static ReportDesign createRowPerPatientXlsOverviewReportDesign(ReportDefinition rd, String resourceName,
-	        String name, Map<? extends Object, ? extends Object> properties) throws IOException {
+	public static ReportDesign createRowPerPatientXlsOverviewReportDesign(ReportDefinition rd, String resourceName, String name, String excelDesignUuid, Map<? extends Object, ? extends Object> properties) throws IOException {
```

Comment:
@rubailly I didn't pick this up last time, but is this meant to be create `RowPerPatient` or create `Indicator` report design?